main.js

```diff
@@ -1,7 +1,36 @@
 import Retry from './src/retry.js'
 
-export default function setup (testFn) {
-  return function retry (name, callback, maxRuns = 2) {
-    return new Retry(name, callback, maxRuns, testFn)
+export default function setup (testFn, defaultMaxRuns = 2) {
```

Comment:
:tada: this looks good, could you mention it in the docs. And the other functions :pray:

Reply:
Updated!

Reply:
thanks